packages/components/src/Menu/Menu.tsx

```diff
@@ -192,10 +257,6 @@ export function Menu({
                 initial="overlayStartStop"
                 animate="done"
                 exit="overlayStartStop"
-                transition={{
-                  type: "tween",
-                  duration: 0.15,
-                }}
```

Comment:
didn't mean to remove this. it was during some animation tweaking. will restore. same for the "floating" change above with the extra types